sites/cablinginstall/server/templates/index.marko

```diff
@@ -161,8 +161,8 @@ $ const { id, alias, name, pageNode } = data;
       </div>
       <div class="col-lg-4 mb-block">
         <marko-web-query|{ nodes }|
-          name="all-published-content"
-          params={ contentTypes: ["Whitepaper"], limit: 3, queryFragment }
+          name="website-scheduled-content"
+          params={ sectionAlias: ["white-papers"], limit: 3, queryFragment }
```

Comment:
The sectionAlias value should be a string, i.e.:

sectionAlias: "white-papers"